starter-code/public/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
looks like `node_modules` was committed to github, maybe before you added it to the `.gitignore`